core/src/table/table.tsx

```diff
@@ -68,6 +72,10 @@ export interface TableProps<R> {
  	*	 when you want to make the table takes 100% of its container width
 	*/
 	fill?: boolean;
+	/**
+	 * A string that describes the table size, which controls table row height.
+	 */
```

Comment:
This is no longer a string, right? Consider the tsdoc of Select.size:

> Size of the select. Choose one from Select.sizes.

Also note that this is applied for table cells, not the cell itself